Pokemon Unity/Assets/Scripts/Save/GlobalSaveManager.cs

```diff
@@ -0,0 +1,275 @@
+﻿//Thanks to Omnicron for the reference to
+// https://pastebin.com/YY0XKMCQ
+//For BinarryFormatting any System.Serializable into a custom file
+
+using System.Runtime.Serialization.Formatters.Binary;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System;
+using UnityEngine.SceneManagement;
+using UnityEngine;
```

Comment:
Dont use "using unityengine" if you're not using `monobehavior`. the entire file will give error in visual studio, and will cause it not to compile. you have to using inline references. example: `UnityEngine.GameObject`